test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -42,21 +41,21 @@
 
 import javax.imageio.ImageIO;
 
-/**
+/*
  * @test
  * @bug 4478780
  * @key headful
  * @summary Tests that Choice can be accessed and controlled by keyboard.
  */
```

Comment:
not sure if this is important but test doesn't have an @<!-- -->run tag which might be good to include

Reply:
@alisenchung `@run` tag is not necessary for automated tests (unless you need to specific other options or run in othervm mode). If the `@run` tag is not specified  jtreg runs the main method of the test class by default.